taskcluster/ci/config.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
This file is a copy of https://github.com/mozilla-mobile/firefox-ios/blob/main/taskcluster/ci/config.yml

Remaning Firefox with Focus